spring-integration-cassandra/src/test/resources/cassandra.properties

```diff
@@ -1,3 +1,3 @@
 cassandra.contactpoints=127.0.0.1
-cassandra.port=9042
-cassandra.keyspace=demo
\ No newline at end of file
+cassandra.port=9043
+cassandra.keyspace=demo
```

Comment:
Isn't related to this PR, but I'm going to revise that all our test stuff and redo it to the proper `@EmbeddedCassandra` with random port.